### ISSUE TITLE
Happy New Year!

### ISSUE THREAD

**Issue by mitming** [lucidrains/magic3d-pytorch#2]
Happy New Year~ (Actually, I came to beg for open source code)

**Comment by fmeres** [comment]
Happy new year! Same here

**Comment by tonymacx86PRO** [comment]
Happy new year! Where is open source code for my rtx 3060?

**Comment by howardgriffin** [comment]
Happy new year! Same here

**Comment by lucidrains** [comment]
oh yes, I should get back to this

do any of you know if https://bluestyle97.github.io/dream3d/ is the new SOTA now? or should I stick with implementing magic3d?

**Comment by lucidrains** [comment]
also, happy new years 🎉

**Comment by cyw-3d** [comment]
Happy new year!I think magic3d is still the sota now. Dream3d can only generate specific categories that in shapenet dataset. Meanwhile, the texture generated by CLIP model, which is used by Dream3d, is less realistic by one’s generated by diffusion model. Therefore, it’s necessary to implement magic3d.发自我的iPhone------------------ Original ------------------From: Phil Wang  ***@***.***>Date: Wed,Jan 11,2023 9:13 AMTo: lucidrains/magic3d-pytorch ***@***.***>Cc: Subscribed ***@***.***>Subject: Re: [lucidrains/magic3d-pytorch] Happy New Year! (Issue #2)
also, happy new years 🎉

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/lucidrains/magic3d-pytorch/issues/2#issuecomment-1378109675",
"url": "https://github.com/lucidrains/magic3d-pytorch/issues/2#issuecomment-1378109675",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by cyw-3d** [comment]
Happy new year!
I think magic3d is still the sota now. Dream3d can only generate specific categories that in shapenet dataset. Meanwhile, the texture generated by CLIP model, which is used by Dream3d, is less realistic by one’s generated by diffusion model. Therefore, it’s necessary to implement magic3d.

**Comment by lucidrains** [comment]
thank you for your expertise! will start looking at this soon then!

**Comment by lucidrains** [comment]
@mitming i'll try my best

i see this work as one step removed away from holodeck, which is simply text to temporal scene representation

**Comment by tonymacx86PRO** [comment]
Please implement **Magic3D**! I will make WebUI for this!

**Comment by AmanKishore** [comment]
Happy New Year! Following this excited to see the progress here, let me know if I can help, have a decent amount of experience diving into Dreamfusion for my app [Mirage](app.mirageml.com)!

**Comment by Jainam213** [comment]
Yeah, using dreamfusion code as a starting point for the coarse model works well.
I agree Magic3d is definately sota!
 I've tried to build a rudimentary implementation myself so let me know if I can help.
A lot of people seem interested to help, it would be a nice idea to create a working group for this

**Comment by yosun** [comment]
Phil, HAPPY NEW YEAR!!!

**Comment by tonymacx86PRO** [comment]
**Dead chat** 💀

**Comment by yosun** [comment]
https://github.com/lucidrains/magic3d-pytorch/issues/3#issuecomment-1546838604
